swagger.yml

```diff
@@ -3556,42 +3608,11 @@ paths:
                     properties:
                       technologies:
                         type: array
+                        description: All available technologies must be returned by default with values at zero if not used.
```

Comment:
This affermation don't seem a description!
@cirelli94 what do you think?

Reply:
I agree we can avoid to specify this!
This is some kind of documentation swagger.yml is not properly suitable, maybe? :thinking:

Reply:
modified the description in "All available technologies that can be monitored by Ercole"

Reply:
:+1:  it's good to me